Reply/app/src/main/res/layout/fragment_email.xml

```diff
@@ -16,128 +16,195 @@
     xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto">
 
-    <data>
-        <variable
-            name="email"
-            type="com.materialstudies.reply.data.Email" />
-    </data>
-
-    <androidx.core.widget.NestedScrollView
-        android:id="@+id/nested_scroll_view"
+  <data>
+    <variable
+        name="email"
+        type="com.materialstudies.reply.data.Email" />
+  </data>
+
+  <androidx.constraintlayout.widget.ConstraintLayout
+      android:layout_width="match_parent"
+      android:layout_height="match_parent"
+      android:layout_marginStart="@dimen/grid_2"
+      android:layout_marginEnd="@dimen/grid_2"
+      android:layout_marginHorizontal="@dimen/grid_0_5"
+      android:layout_marginVertical="@dimen/grid_4">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
I think this ConstraintLayout, which you're using to create the toolbar at the top of the screen, would be better as an AppBarLayout+Toolbar so when the email replies scroll, it can remain at the top of the screen

Reply:
Used a constrainlayout within this hope it is okay.